lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -1290,17 +1395,31 @@ def rotate_axes(xs, ys, zs, zdir):
         return xs, ys, zs
 
 
-def _zalpha(colors, zs):
+def _zalpha(
+    colors,
+    zs,
+    min_alpha=0.3,
+    _data_scale=None,
+):
```

Comment:
Should fit:
```suggestion
def _zalpha(colors, zs, min_alpha=0.3, _data_scale=None):
```
Also, the function is already private, so not sure why `_data_scale` is too?

Reply:
> Also, the function is already private, so not sure why `_data_scale` is too?

Ping here?

Reply:
updated